tests/ui-api-layer-acceptance-tests/graphql/client.go

```diff
@@ -39,6 +39,7 @@ func New() (*Client, error) {
 
 	httpClient := newAuthorizedClient(token, clusterContext)
 	gqlClient := graphql.NewClient(config.GraphQlEndpoint, graphql.WithHTTPClient(httpClient))
+	//gqlClient.Log = func(s string) { log.Println(s) }
```

Comment:
Remove comment

Reply:
BTW I understand the intentions of using custom logger for GraphQL clien. You can log all messages to some variable and print the value only when something goes wrong

Reply:
Comment was only for testing, but I at the moment logs only are show only for specific failed query.